### PR TITLE
feat: update pages head attribute

### PR DESCRIPTION
**Pull request by boojack**
Update all page head attribute with metadata for SEO.

### CODE REVIEW

**Review by d-bytebase — APPROVED** [review]

**Review by tianzhou — APPROVED** [review]
LGTM

File: pages/pricing/index.vue [hunk 1]
```diff
@@ -551,6 +551,19 @@ export default defineComponent({
...
   },
+  head() {
+    return {
+      title: "Bytebase Pricing",
+      meta: [
+        {
+          hid: "Bytebase Pricing",
+          name: "Bytebase Pricing",
+          content:
+            "Free, Personal project or small team, no DBA. Team, Medium size team, with dedicated DBA or TL for engineering velocity. You can upgrade, downgrade, or cancel  your subscription anytime. No hidden charges.",
```
Comment (tianzhou) [inline pages/pricing/index.vue:562]:
```suggestion
            "Free to start. Paid plan available. Cancel anytime. No hidden charges.",
```

**Review by tianzhou — APPROVED** [review]
LGTM

File: pages/pricing/index.vue (hunk 1 above)
Comment (boojack) [reply pages/pricing/index.vue:562]: Check out the separated PR: https://github.com/bytebase/bytebase.com/pull/109

**Pull request merged**